packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -45,32 +44,31 @@ export interface IContextualMenuProps extends IBaseProps<IContextualMenu>, IWith
   styles?: IStyleFunctionOrObject<IContextualMenuStyleProps, IContextualMenuStyles>;
 
   /**
-   * Theme provided by High-Order Component.
+   * Theme provided by higher-order component.
```

Comment:
No actual changes to the interfaces in this file, just comments (I did a good bit of reading the comments while trying to figure out how to fix the styling issue, and decided to edit them)